spec/statesman/machine_spec.rb

```diff
@@ -615,8 +615,12 @@ def after_initialize; end
 
     context "when it is unsuccesful" do
       before do
+        guard_failed_error = Statesman::GuardFailedError.new(
+          "Guard on transition from: 'active' to 'inactive' returned false"
+        )
+
         allow(instance).to receive(:transition_to!).
-          and_raise(Statesman::GuardFailedError)
```

Comment:
Going to experiment with this today - it seems pretty weird that it breaks it. Wondering if not using a keyword argument might help.